multiversx/mx-chain-go
# Elasticsearch support for testnet scripts

## Reasoning behind the pull request
- testnet scripts didn't have support for indexing data into Elasticsearch
  
## Proposed changes
- added support for Elasticsearch, which starts an ES docker container

## Testing procedure
- set `USE_ELASTICSEARCH` to `1` inside `variables.sh` and start the testnet. It should initiate an Elasticsearch container running on the `9200` port and it should be closed when `./stop.sh` is called

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
